webanno/webanno
# #1355 Add a setting to hide the usernames from the dropdown list

**What's in the PR**
webanno#1355 - Add a setting to hide the usernames from the dropdown list.
Reads from the settings.properties file the configuration settings userSelection.hideUsers, userSelection.minLengthCharacters and modifies the behavior of the dropdown list accordingly.
Updates the user guide User section.
Introduces the new settings to the admin guide General Settings table.

**How to test manually**
Add the userSelection.hideUsers=true setting into the settings.properties file.
Go to the Users tab and click into the textField to add a user.  You shouldn't see any dropdown list with users.
Write the username of the user you wish to add. Only this username should be visible in the dropdown list.

**Documentation**
